src/Test/Perf/Runner/Program.cs

```diff
@@ -20,8 +20,28 @@ private static bool LaunchedWithArgument(string arg)
             return Environment.GetCommandLineArgs().Contains($"/{arg}") ||
                    Environment.GetCommandLineArgs().Contains($"--{arg}");
         }
+
+        private static string ValueForCommandLineKey(string key, string otherwise=null)
+        {
+            var args = Environment.GetCommandLineArgs().ToList();
+            var n = args.IndexOf($"/{key}");
+            n = (n == -1) ? args.IndexOf("--{key}") : n;
+
+            if (n == -1)
+            {
+                return otherwise;
+            }
+
+            if (args.Count == n)
```

Comment:
This code is removed now, but you were right, it should have been "args.Count - 1 == n"